docs/sources/enterprise/license/activate-license.md

```diff
@@ -17,35 +18,50 @@ To download your Grafana Enterprise license:
 1. Go to **My Account** and select an organization from the drop-down menu at the top left of the page. On the Overview page for each organization, you can see a section for Grafana Enterprise licenses. Click **Details** next to a license.
 1. At the bottom of the license details page, select **Download token** to download the license.jwt file that contains your license.
 
-## Step 2. Add your license file to a Grafana instance
+## Step 2. Add your license to a Grafana instance
 
-There are two different ways to add the license file to a Grafana instance:
+There are three different ways to add the license to a Grafana instance:
```

Comment:
Let's generalize this and remove the baked in number:
```suggestion
There is more than one way to add the license to a Grafana instance:
```

Reply:
Added manually. (Had messed up the first merge conflict so couldn't apply the suggestion automatically)